src/test/java/com/yahoo/bullet/rest/service/QueryServiceTest.java

```diff
@@ -60,4 +83,33 @@ public void testSubmit() throws Exception {
         Mockito.verify(publisher).send(randomID, randomContent);
         service.close();
     }
+
+    @Test
+    public void testSubmitSignal() throws Exception {
+        Publisher publisher = Mockito.mock(Publisher.class);
+        Subscriber subscriber = new MockSubscriber();
+        QueryHandler queryHandler = Mockito.mock(QueryHandler.class);
+        QueryService service = new QueryService(singletonList(publisher), singletonList(subscriber), 1);
+        service.getRunningQueries().put("id", queryHandler);
+
+        service.submitSignal("id", Metadata.Signal.KILL);
+        Assert.assertEquals(0, service.getRunningQueries().size());
+        Mockito.verify(publisher).send(
```

Comment:
Move send to new line and indent with verify

Reply:
Done.